modules/manage/partials/tiered-storage/gcp-kms-key.adoc

```diff
@@ -0,0 +1,68 @@
+[discrete]
+= Configure access to data in GCP using a KMS key
+
+To configure the Google Cloud bucket used by Redpanda Tiered Storage to leverage a customer-managed key using the Cloud Key Management Service API instead of the default Google-managed key, you must:
+. Create a KMS key.
+. Configure the bucket to use the KMS key.
+. Optionally, re-encrypt existing data with the new KMS key.
+To manage Google Cloud KMS using the command line, first install or upgrade to the latest version of https://cloud.google.com/sdk/docs/install[Google Cloud CLI^].
+To create a KMS key:
+. In Google Cloud Console, search for "Cloud Key Managment Service API". Click **Enable** if the service is not already enabled.
+. Using the Google Cloud CLI, create a new keyring in the https://cloud.google.com/kms/docs/locations^[region] where the bucket used by Redpanda is located. Note that region is case-sensitive.
++
```

Comment:
This isn't rendering properly. See https://deploy-preview-808--redpanda-docs-preview.netlify.app/24.3/manage/tiered-storage/?tab=tabs-1-google-cloud-storage